bindata/manifests/metrics-exporter/metrics-daemonset.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app.kubernetes.io/name: sriov-network-metrics-exporter
+  name: sriov-network-metrics-exporter
+  namespace: {{.Namespace}}
+  annotations:
+    release.openshift.io/version: "{{.ReleaseVersion}}"
```

Comment:
why we need this one?

Reply:
No idea TBH. All the daemonset has that annotation but it doesn't seem to have particular value. Removing